docs/devGuide/githubActions/markbindAction.md

```diff
@@ -35,14 +35,14 @@ The source code is at [MarkBind/markbind-action](https://github.com/MarkBind/mar
 
 ## Testing the action
 1. Include a workflow file in your <tooltip content="Which includes a sample MarkBind site">test repository</tooltip> to test your modified version of the action.
-   * Instead of using `uses: MarkBind/markbind-action@v2` in the workflow file, use `uses: yourGitHubName/markbind-action@yourBranch` to reference the unpublished version of the action that you are currently developing.
+   * Instead of using `uses: MarkBind/markbind-action@v2` in the workflow file, use `uses: yourGithubName/markbind-action@yourBranch` to reference the unpublished version of the action that you are currently developing.
```

Comment:
Sorry just one last qn, for this change is it accidental?

Reply:
Hmm, I did it on purpose because I thought camelCase would look better if only the first letter of each word is in uppercase. I think this is contentious though, which do you think is better? We seem to use both ways, e.g. `userGuide/usingHtmlJavaScriptCss.md` (JavaScript and not Javascript, but Html and not HTML), `testMarkbindPluginScript.js` (Markbind and not MarkBind)

Reply:
I am ok with either. The impact is quite small in this specific case since it's not a variable or a file name.

> e.g. userGuide/usingHtmlJavaScriptCss.md (JavaScript and not Javascript, but Html and not HTML), testMarkbindPluginScript.js (Markbind and not MarkBind)

I guess for these examples...would be good to be consistent. If we do want to standardize then perhaps we should check if there are other instances and fix those as well.